include/queue/helper.hpp

```diff
@@ -20,14 +20,14 @@ inline buffer_iterator<scalar_t> make_sycl_iterator_buffer(
   using buff_t =
       blas::buffer_t<scalar_t, 1, cl::sycl::default_allocator<scalar_t>>;
   return blas::buffer_iterator<scalar_t>{
-      buff_t{data.data(), cl::sycl::range<1>{size}}};
+      buff_t{data.data(), cl::sycl::range<1>(size)}};
 }
 
 template <typename scalar_t, typename index_t>
 inline blas::buffer_iterator<scalar_t> make_sycl_iterator_buffer(index_t size) {
   using buff_t =
       blas::buffer_t<scalar_t, 1, cl::sycl::default_allocator<scalar_t>>;
-  return blas::buffer_iterator<scalar_t>{buff_t{cl::sycl::range<1>{size}}};
+  return blas::buffer_iterator<scalar_t>{buff_t{cl::sycl::range<1>(size)}};
```

Comment:
isn't this giving you a new warning about conversions?

Reply:
No-This will implicitly convert with no warning. However, the new c++ feature give me error. Since we force the range type to be size_t, I can either static cast or use the old style approach.